.build.ps1

```diff
@@ -202,10 +202,10 @@ task buildDocs -Inputs $bdInputs -Outputs $bdOutputs {
     Copy-Item -Path $docsPath\about_PSScriptAnalyzer.help.txt -Destination $outputDocsPath -Force
 
     # Build documentation using platyPS
-    if ((Get-Module PlatyPS -ListAvailable) -eq $null) {
-        throw "Cannot find PlatyPS. Please install it from https://www.powershellgallery.com."
+    if ($null -eq (Get-Module platyPS -ListAvailable -Verbose:$verbosity | Where-Object { $_.Version -ge 0.9 })) {
+        throw "Cannot find platyPS of version greater or equal to 0.9. Please install it from https://www.powershellgallery.com/packages/platyPS/ using e.g. the following command: Install-Module platyPS"
     }
-    Import-Module PlatyPS
+    Import-Module platyPS
```

Comment:
should this have `-minimumversion 0.9.0`?

Reply:
I thought this is not needed because we already checked on the line above that the user has a module with a version greater or equal to 0.9. Does Import-Module not always import the latest version in some case?
Would it in general be better to require that 0.9.x is exactly installed and not allow newer versions at all? Then we would need to use the `-RequiredVersion` parameter. What do you think?

Reply:
Yes, it will import the latest version and there's a later test to be sure it meets the requirement. It's probably fine, just wanted to be sure.